### ISSUE TITLE
Scaling issues when main screen attached

### ISSUE THREAD

**Issue by xperiandri** [UnigramDev/Unigram#1043]
![image](https://user-images.githubusercontent.com/2365592/55620704-2d6ecd00-57a4-11e9-9e78-ba9ea416d73e.png)

When I attach 1920x1080 (125% scale) monitor (set as main) to my 2736x1824 (200% scale) Surface Pro the issue on screenshot above happens.
This does not fix untill I relaunch whole Ungram app.

The similar behavior present if app is launched on monitor and then it is detached

**Comment by FrayxRulez** [comment]
I'm not sure this is an app bug as the app has no way to manage how scale is applied

**Comment by xperiandri** [comment]
Other apps scale without this issue

**Comment by FrayxRulez** [comment]
Can you provide me an external screen to test? :D

**Comment by xperiandri** [comment]
😃
I would've like to do that.
Do you go to coworkings from time to time?

**Comment by xperiandri** [comment]
Do you have any clue where to look? I will try to look myself

**Comment by FrayxRulez** [comment]
No, I have absolutely no idea as I told you I'm not handling DPI changes as Windows should do this for me (nor I prevent default behavior)

**Comment by MarcAnt01** [comment]
Marked as blocked since Fela is missing the required equipment to reproduce the bug

**Comment by FrayxRulez** [comment]
Any luck?

**Comment by xperiandri** [comment]
I'll check next week

**Comment by FrayxRulez** [comment]
This is a bug of the UWP platform, I seen this happen with other apps as well. So not related to Unigram: closing this.
